sdks/typescript/packages/sdk/src/db_connection_builder.ts

```diff
@@ -20,6 +20,7 @@ export class DbConnectionBuilder<
   #emitter: EventEmitter<ConnectionEvent> = new EventEmitter();
   #compression: 'gzip' | 'none' = 'gzip';
   #lightMode: boolean = false;
+  #confirmedReads: boolean = false;
```

Comment:
Can we make this `boolean?`, so that we can set a query param if and only if the user sets a value? That way we could let the server decide what the default is.

Reply:
Ah, the question mark. Such concise.